### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by LucasSD**
Add PostgreSQL database and development setup instructions for Windows users, including some small edits to existing instructions.

### CODE REVIEW

**Comment by LucasSD** [comment]
I tried to include enough detail to make the instructions easy to follow for new contributors using Windows. I'm not sure I've handled the database settings in the best way, but it worked for me. Let me know what you think @chriswedgwood

File: README.rst [hunk 1]
```diff
@@ -3,8 +3,37 @@ djangosnippets.org
...
+Download the latest version of PostgreSQL_. Click on the executable to start the installation setup wizard.
+
+Click ``Next``, keeping all the defaults, ensuring that all components are ticked (including pgAdmin 4). Make a note
+of the password you choose for the database superuser (postgres). Select the default port 5432 and the default locale.
+After it’s finished installing, you do not need to launch Stack Builder. Un-tick that box if you are asked, and
+click ``Finish``.
+
+Open the pgAdmin 4 app. Type in the password you made a note of earlier. The following steps can also be taken using
+your windows CLI: Click on ``Servers`` in the Browser pane on the left. Right click on ``PostgreSQL`` and then click
+on ``Create``, then ``Database``. In the Database field which you should now see, type 'djangosnippets', then click
```
Comment (chriswedgwood) [inline README.rst:18]: It might be better to just use powershell or cmd to work with postgres as that will eliminate the dependency on pgAdmin 4.

**Review by chriswedgwood — COMMENTED** [review]
> I tried to include enough detail to make the instructions easy to follow for new contributors using Windows. I'm not sure I've handled the database settings in the best way, but it worked for me. Let me know what you think @chriswedgwood

Thanks for the contributions :)
Ive added a couple comments. Let me know if they make sense.

**Comment by chriswedgwood** [comment]
Also if it will help Im happy to get on a zoom/teams call to discuss @LucasSD

**Comment by LucasSD** [comment]
> Also if it will help Im happy to get on a zoom/teams call to discuss @LucasSD

Thanks Chris. I should be able to get this sorted soon. If I run into any blocks, I'll let you know and we can jump on a call.

**Comment by LucasSD** [comment]
> Also if it will help Im happy to get on a zoom/teams call to discuss @LucasSD

Hi @chriswedgwood . I have a better solution for Windows setup now, but would be good to run it by you first before I change the pr. I'll drop you an e-mail

**Comment by LucasSD** [comment]
Hi @chriswedgwood. I've updated the instructions in line with your comments. As far as I know, there is no Windows equivalent to the source command. The best solution seems to be to create a batch file. I think it's simplest to list the contents of the file for new users, instead of describing each alteration. Let me know what you think.

**Comment by chriswedgwood** [comment]
Cool lets take this and if we need to we can improve it. Ironically I left  a job last week that I was using windows so I cant play around and see if there is a way to use .env with windows so perhaps we didn't need env.bat. But if you are happy that it works for you then I am happy :). Nice work

**Comment by LucasSD** [comment]
> Cool lets take this and if we need to we can improve it. Ironically I left a job last week that I was using windows so I cant play around and see if there is a way to use .env with windows so perhaps we didn't need env.bat. But if you are happy that it works for you then I am happy :). Nice work

It definitely works and others have used that approach. There is a package called python-dotenv which would enable the use of .env, but at the cost of an additional dependency, and perhaps a little more complexity for new users. Cheers

**Pull request merged**